src/main/kotlin/io/zerobase/smarttracing/features/devices/DevicesDao.kt

```diff
@@ -87,4 +89,49 @@ class DevicesDao @Inject constructor(private val graph: GraphTraversalSource) {
             throw EntityCreationException("Error creating scan relationship between devices.", ex)
         }
     }
+
+    fun recordTestResult(testResult: TestResult): ReportId {
+        val reportId = randomUUID().toString()
+        try {
+            val deviceNode = graph.V(testResult.testedParty.value).getIfPresent() ?: throw InvalidIdException(testResult.testedParty)
+            val reporterNode = graph.V(testResult.reportedBy.value).getIfPresent() ?: throw InvalidIdException(testResult.reportedBy)
+            graph.addV("TestResult").
+                property(T.id, reportId).
+                property(single, "verified", testResult.verified).
```

Comment:
I didn't see an endpoint to update this verified boolean -- it's probably not urgent but thought it might be good in case someone returns and wants to change their verification. We'd probably also want to check who can verify this? It's probably in the issues (haven't taken a look) or I overlooked it in these changes

Reply:
you can't verify self-reported symptoms. the only thing of symptoms that can be verified would be the temperature really. i set it up this way so we could, in theory, have healthcare professionals inputing symptom reports and we would consider their reports "verified", while a self-report is not.